templates/al2/provisioners/upgrade-kernel.sh

```diff
@@ -5,7 +5,7 @@ set -o nounset
 set -o errexit
 
 if [[ -z "$KERNEL_VERSION" ]]; then
-  if vercmp "$KUBERNETES_VERSION" gteq "1.24.0"; then
+  if vercmp "$KUBERNETES_VERSION" gteq "1.23.0"; then
     KERNEL_VERSION=5.10
   else
     KERNEL_VERSION=5.4
```

Comment:
should include hits as default even for below 1.23?
```suggestion
  KERNEL_VERSION=5.10
```

Reply:
Yes, include 1.21 and we need another new AMI 😢